### PR TITLE
updated recode_ATPT.R and pk_variables.R-

### PR DESCRIPTION
**Pull request by SwagataBhattacharyya**
recode_ATPT(): Update to code for pattern “18 TO 24 Hours” and “-18 TO -24 Hours” which takes the larger value for recoding; added ATPT_mod for removal of brackets prior to derivation
pk_variables(): Update to code for recoding of pattern “>400” to AVALN=0

### CODE REVIEW

**Comment by billdenney** [comment]
My guess is that this is an issue with the new version of dplyr.  Can you please try installing the current version of dplyr and see if you can replicate it locally?

**Comment by SwagataBhattacharyya** [comment]
@billdenney- Thank you for the pointer. Reinstalling the current version of dplyr highlighted the errors. Also, I can now confirm that there are no more instances of suppressWarnings() in any of the functions in this branch. 

This branch is ready for review.

File: R/check_units.R [hunk 1]
```diff
@@ -30,8 +30,8 @@ check_units <- function(data, col1, col2, exclude_var) {
     distinct() %>%
     ungroup() %>%
     pivot_wider(names_from = "STUDYID", values_from = "units", values_fn = list) %>%
-    suppressWarnings(unnest()) %>%
-    suppressWarnings(mutate_each(funs(replace(., is.na(.), ""))))
+    unnest(cols=c(2,3)) %>%
```
Comment (billdenney) [inline R/check_units.R]: Instead of using column numbers, can you please use column names?

**Review by billdenney — COMMENTED** [review]
Can you please fix that one note about column names?  Then it will be ready to merge.

**Comment by billdenney** [comment]
Thanks!

**Pull request merged**